contrib/mesos/pkg/scheduler/service/service.go

```diff
@@ -616,14 +624,8 @@ func (s *SchedulerServer) awaitFailover(schedulerProcess schedulerProcessInterfa
 
 func validateLeadershipTransition(desired, current string) {
 	log.Infof("validating leadership transition")
-	d := uid.Parse(desired).Group()
-	c := uid.Parse(current).Group()
-	if d == 0 {
-		// should *never* happen, but..
-		log.Fatalf("illegal scheduler UID: %q", desired)
-	}
-	if d != c && c != 0 {
-		log.Fatalf("desired scheduler group (%x) != current scheduler group (%x)", d, c)
+	if desired != current && current != "" {
```

Comment:
I probably can't since `desired` is being fetched from etcd, and we probably don't ever write "" into etcd. I think the old code checked it for sanity.